evm_loader/cli/src/account_storage.rs

```diff
@@ -220,14 +220,7 @@ impl<'a> EmulatorAccountStorage<'a> {
                 eprintln!("account key:  {}", &solana_address.to_string());
                 eprintln!("code account: {}", &account_data.code_account.to_string());
 
-                if let Some(acc) = config.rpc_client.get_account_with_commitment(&account_data.code_account, CommitmentConfig::processed()).unwrap().value {
-                    eprintln!("Account found");
-                    Some(acc)
-                }
-                else {
-                    eprintln!("Account not found");
-                    None
-                }
+                config.rpc_client.get_account_with_commitment(&account_data.code_account, CommitmentConfig::processed()).unwrap().value
```

Comment:
I just interested if someone guarantees safety unwrapping and handling a `ClientError`

Reply:
No guarantees. It will panic if there was a network error.
But it does not really matter